sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_base_client.py

```diff
@@ -13,8 +13,9 @@
 class TextAnalyticsApiVersion(str, Enum):
     """Text Analytics API versions supported by this package"""
 
+    #V3_2_PREVIEW = "v3.2-preview.1"
```

Comment:
tiny nit: we can just remove this comment

Reply:
Ok, I removed this and pushed the change but for some reason I'm not seeing the change reflected in the PR.  I'll leave this open for now until I can figure out why I'm not seeing it.

Reply:
Nvm, I'm just too used to the representation in Azure Devops.  Looks like the change is here, it's just not shown in the context of this comment.